src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -218,6 +220,8 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
             The random seed to set for subsetting
         channel_percentile (float):
             Percentile used to normalize channels to same range
+        channel_percentile_postnorm (float):
+            Percentile used for second round of normalization
```

Comment:
This is very inconsequential, but to make it a tad more clear, can we change this description to say "Percentile used to normalize channels after pixel normalization"?